packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -906,15 +906,15 @@ async function handler(
     return deletedReferences;
   };
 
-  const handleSeats = async (): Promise<
-    | (Partial<Booking> & {
-        appsStatus?: AppsStatus[];
-        seatReferenceUid?: string;
-        paymentUid?: string;
-        message?: string;
-      })
-    | null
-  > => {
+  const handleSeats = async () => {
+    let resultBooking:
+      | (Partial<Booking> & {
+          appsStatus?: AppsStatus[];
+          seatReferenceUid?: string;
+          paymentUid?: string;
+          message?: string;
+        })
+      | null = null;
```

Comment:
Changes here are: Add some logic so we only have one return so we can run workflows and everything else in every case.